packages/ckeditor5-font/tests/ui/colortableview.js

```diff
@@ -101,767 +89,6 @@ describe( 'ColorTableView', () => {
-					documentColorsGrid = dropdown.colorTableView.colorGridsPageView.documentColorsGrid;
-				} );
-		} );
-
-		afterEach( () => {
-			element.remove();
-			dropdown.element.remove();
-			dropdown.destroy();
-
-			return editor.destroy();
-		} );
-
-		it( 'should have color tile turned on in document colors and static colors section', () => {
-			const command = editor.commands.get( 'testColorCommand' );
-
-			setModelData( model,
-				'<paragraph><$text testColor="red">Foo</$text></paragraph>'
-			);
-			command.value = 'red';
-
-			dropdown.isOpen = true;
-
-			expect( staticColorsGrid.selectedColor ).to.equal( 'red' );
-			expect( documentColorsGrid.selectedColor ).to.equal( 'red' );
-
-			const redStaticColorTile = staticColorsGrid.items.find( tile => tile.color === 'red' );
-			const redDocumentColorTile = documentColorsGrid.items.get( 0 );
-
-			expect( redStaticColorTile.isOn ).to.be.true;
-			expect( redDocumentColorTile.isOn ).to.be.true;
-		} );
-
-		it( 'should have color tile turned on in static colors section when document colors are full', () => {
-			const command = editor.commands.get( 'testColorCommand' );
-
-			setModelData( model,
-				'<paragraph><$text testColor="gold">Bar</$text></paragraph>' +
-				'<paragraph><$text testColor="rgb(10,20,30)">Foo</$text></paragraph>' +
-				'<paragraph><$text testColor="#FFAACC">Baz</$text></paragraph>' +
-				'<paragraph><$text testColor="#00FF00">Test</$text></paragraph>'
-			);
-			command.value = '#00FF00';
-
-			dropdown.isOpen = true;
-
-			expect( staticColorsGrid.selectedColor ).to.equal( '#00FF00' );
-			expect( documentColorsGrid.selectedColor ).to.equal( '#00FF00' );
-
-			const redStaticColorTile = staticColorsGrid.items.find( tile => tile.color === '#00FF00' );
-			const activeDocumentColorTile = documentColorsGrid.items.find( tile => tile.isOn );
-
-			expect( redStaticColorTile.isOn ).to.be.true;
-			expect( activeDocumentColorTile ).to.be.undefined;
-		} );
-
-		it( 'should not have a selection for unknown colors exceeding document colors limit', () => {
-			const command = editor.commands.get( 'testColorCommand' );
-
-			setModelData( model,
-				'<paragraph><$text testColor="gold">Bar</$text></paragraph>' +
-				'<paragraph><$text testColor="rgb(10,20,30)">Foo</$text></paragraph>' +
-				'<paragraph><$text testColor="#FFAACC">Baz</$text></paragraph>' +
-				'<paragraph><$text testColor="pink">Test</$text></paragraph>'
-			);
-			command.value = 'pink';
-
-			dropdown.isOpen = true;
-
-			expect( staticColorsGrid.selectedColor ).to.equal( 'pink' );
-			expect( documentColorsGrid.selectedColor ).to.equal( 'pink' );
-
-			const activeStaticDocumentTile = staticColorsGrid.items.find( tile => tile.isOn );
-			const activeDocumentColorTile = documentColorsGrid.items.find( tile => tile.isOn );
-
-			expect( activeStaticDocumentTile ).to.be.undefined;
-			expect( activeDocumentColorTile ).to.be.undefined;
-		} );
-	} );
-
 	describe( 'disabled document colors section', () => {
```

Comment:
What made you leave these tests behind in ckeditor5-font? Shouldn't they land in ckeditor5-ui?